src/ResourceStore.php

```diff
@@ -28,6 +28,8 @@ class ResourceStore
 
     protected $resources = [];
 
+    protected $disable_to_many_replace = false;
```

Comment:
This is a double negative. Suggest renaming as a positive:

`$is_to_many_replace_enabled`